### PR TITLE
welcome page upgrade

### PR DESCRIPTION
**Pull request by sonylomo**
Fixes issue #101 
- Changed Icons component to pf4.
- Destructured some props.

### CODE REVIEW

**Review by jennydaman — CHANGES_REQUESTED** [review]
Looks good for the most part. However, for this project we're using `yarn` not `npm` so please

```shell
git rm package-lock.json
```

File: src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './WelcomeFeature.css';
 
-const WelcomeFeature = props => (
+const WelcomeFeature = ({img, name}) => (
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js:5]: As a practice it's good to also accept

```javascript
const WelcomeFeature = ({img, name, ...props}) => (
```

And then spread `{ ...props}` on the parent element returned.

**Review by jennydaman — CHANGES_REQUESTED** [review]

File: src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js (hunk 1 above)
Comment (sonylomo) [reply src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js]: I realized that I hadn't made changes to `WelcomeCategories` component. I've also implemented the changes you requested.

File: src/components/Welcome/components/WelcomeCategory/WelcomeCategory.js [hunk 2]
```diff
@@ -1,29 +1,23 @@
 import React from 'react';
-import { Card, CardTitle, CardBody, CardFooter } from 'patternfly-react';
+import { Card, CardTitle, CardBody, CardFooter } from "@patternfly/react-core";
 import Button from '../../../Button';
 import PropTypes from 'prop-types';
 import './WelcomeCategory.css';
 
-const WelcomeCategory = props => (
+const WelcomeCategory = ({ category, ...props }) => (
   <Card className="welcome-category">
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeCategory/WelcomeCategory.js:8]: Consider adding a wrapper `<div {...props}>`

File: src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js [hunk 3]
```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './WelcomeFeature.css';
 
-const WelcomeFeature = props => (
+const WelcomeFeature = ({img, name, ...props}) => (
   <div className="welcome-feature">
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeFeature/WelcomeFeature.js:6]: Consider adding a wrapper `<div {...props}>`

**Review by jennydaman — CHANGES_REQUESTED** [review]
For flexibility, the parent returned by `render` should be attribute-less and spread `{...props}` so that attributes can be passed from the client. Usually useful for setting a class.

**Review by jennydaman — CHANGES_REQUESTED** [review]
Spacing between elements and responsive centering is missing. Could you take a look?

screenshot

**Comment by sonylomo** [comment]
Sorry I missed that - it's coz of the added <div> wrapper.

**Review by jennydaman — APPROVED** [review]

**Pull request merged**